app/controllers/people/sac_family_main_person_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+class People::SacFamilyMainPersonController < ApplicationController
+  def update
+    authorize!(:update, person)
+
+    unless person.sac_family_member?
+      return render plain: 'Person is not associated with any household',
+                    status: :unprocessable_entity
+    end
```

Comment:
machts vielleicht Sinn die conditional redirects/renders in eine jeweils sprechende Methode zu packen. Dann wäre das etwas leserlicher
Sowas wie `def assert_person_family_member` & `def assert_already_main_family_person`

Einfach um den noise in dieser Methode etwas zu reduzieren